common/navbar.js

```diff
@@ -44,13 +51,36 @@
             scope: {},
             templateUrl: '../common/templates/navbar.html',
             link: function(scope) {
+              if(chaiseConfig.navbarBrand)
```

Comment:
The rest of the checks below can be removed because of `setConfigJSON()`. Each property should be set like the following:
```js
scope.<property> = scope.$root.chaiseConfig.<property>
```